quoridorcore/src/main/java/com/harush/zitoon/quoridor/core/model/Board.java

```diff
@@ -108,7 +112,7 @@ public boolean containsWall(int x, int y, boolean isHorizontal) {
     }
 
     public boolean containsPawn(int x, int y) {
```

Comment:
In Java, boolean properties' are by convention named with 'is'.
It's bad practice to break standard convention.

Reply:
Perhaps I wasn't clear enough. It's not specifically `is`. It should form a question:
[](https://stackoverflow.com/questions/3874350/naming-conventions-for-java-methods-that-return-booleanno-question-mark)